docs/gettingstarted/installer.md

```
@@ -1,52 +1,148 @@
+![Dig2Go](https://shop.allnetchina.cn/cdn/shop/products/Led_4.jpg?v=1680836018&width=1600){: style="width:250px"} 
+
+Connect the device via a USB cable (which supports data transfer) to your PC.
+
+## Flash the firmware
 
 <script type="module" src="https://unpkg.com/esp-web-tools@10/dist/web/install-button.js?module"></script>
 
-MoonLight v0.7.0, 22 December 2025
+Select your ESP32-device from the list below and press the corresponding Connect button and follow these steps:
+
+<img width="250" src="https://github.com/user-attachments/assets/f092743f-a362-40da-b932-d31b203d966d" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)